scripts/release.sh

```diff
@@ -146,30 +146,37 @@ if ! [[ $create =~ ^[Yy]$ ]]; then
 	exit 0
 fi
 
-args=()
+# Set safe default values.
+draft_json=false
+dry_run_json=true
+ignore_missing_commit_metadata_json=false
+
 if ((draft)); then
-	args+=(-F draft=true)
-else
-	args+=(-F draft=false)
+	draft_json=true
 fi
-if ((dry_run)); then
-	args+=(-F dry_run=true)
-else
-	args+=(-F dry_run=false)
+if ! ((dry_run)); then
+	dry_run_json=false
 
 	# We only set this on non-dry-run releases because it will show a
 	# warning in CI.
 	if [[ ${CODER_IGNORE_MISSING_COMMIT_METADATA:-0} == 1 ]]; then
-		args+=(-F ignore_missing_commit_metadata=true)
+		ignore_missing_commit_metadata_json=true
 	fi
 fi
 
+gh_json_input() {
+	cat <<EOJ
+{
+	"increment": "$increment",
+	"draft": $draft_json,
+	"dry_run": $dry_run_json,
+	"snapshot": false
+}
+EOJ
+}
+
 log
-gh workflow run release.yaml \
-	--ref "$branch" \
-	-F increment="$increment" \
-	-F snapshot=false \
-	"${args[@]}"
+gh_json_input | gh workflow run release.yaml --ref "$branch"
```

Comment:
Why not just `gh_json_input="..."` followed by `echo "$gh_json_input" | gh ...`?

Reply:
Mainly because I didn't want to do a bunch of escapes on the quotes (`\"`) when writing the json.

Reply:
Using `jq` would've been sensible here.